doc/classes/MultiMesh.xml

```diff
@@ -82,32 +84,45 @@
 	</methods>
 	<members>
 		<member name="color_format" type="int" setter="set_color_format" getter="get_color_format" enum="MultiMesh.ColorFormat">
+			Format of colors in color array that gets passed to shader.
 		</member>
 		<member name="custom_data_format" type="int" setter="set_custom_data_format" getter="get_custom_data_format" enum="MultiMesh.CustomDataFormat">
+			Format of custom data in custom data array that gets passed to shader.
 		</member>
 		<member name="instance_count" type="int" setter="set_instance_count" getter="get_instance_count">
+			Number of instances that will get drawn.
 		</member>
 		<member name="mesh" type="Mesh" setter="set_mesh" getter="get_mesh">
+			Mesh to be drawn.
 		</member>
 		<member name="transform_format" type="int" setter="set_transform_format" getter="get_transform_format" enum="MultiMesh.TransformFormat">
+			Format of transform used to transform mesh, either 2D or 3D.
 		</member>
 	</members>
 	<constants>
 		<constant name="TRANSFORM_2D" value="0" enum="TransformFormat">
+			Use this when using 2D transforms.
 		</constant>
 		<constant name="TRANSFORM_3D" value="1" enum="TransformFormat">
+			Use this when using 3D transforms.
 		</constant>
 		<constant name="COLOR_NONE" value="0" enum="ColorFormat">
+			Use when you are not using per-instance [Color]s.
 		</constant>
 		<constant name="COLOR_8BIT" value="1" enum="ColorFormat">
+			Compress [Color] data into 8 bits when passing to shader. This uses less memory and can be faster, but the [Color] loses precision.
 		</constant>
 		<constant name="COLOR_FLOAT" value="2" enum="ColorFormat">
+			The [Color] passed into [set_instance_color] will use 4 floats. Use this for highest precision [Color].
 		</constant>
 		<constant name="CUSTOM_DATA_NONE" value="0" enum="CustomDataFormat">
+			Use when you are not using per-instance custom data.
 		</constant>
 		<constant name="CUSTOM_DATA_8BIT" value="1" enum="CustomDataFormat">
+			Compress custom_data into 8 bits when passing to shader. This uses less memory and can be faster, but loses precision.
 		</constant>
 		<constant name="CUSTOM_DATA_FLOAT" value="2" enum="CustomDataFormat">
+			The [Color] passed into [set_instance_custom_data] will use 4 floats. Use this for highest precision.
```

Comment:
Method references are done with `[method <method_name>]`, so it should be `[method set_instance_custom_data]`.